sdk/eventhub/Azure.Messaging.EventHubs.Processor/tests/EventProcessorClient/EventProcessorClientOptionsTests.cs

```diff
@@ -20,6 +21,7 @@ public class EventProcessorClientOptionsTests
         /// </summary>
         ///
         [Test]
+        [Ignore("Cannot access RetryOptions.IsEquivalentTo in the main package.")]
```

Comment:
Same here. This is not that simple to fix. We'll probably need to move some stuff around in the shared package.